includes/classes/Jobber.php

```diff
@@ -99,7 +139,8 @@ protected function query( string $form_type = '', bool $force = false ) {
 		];
 
 		// Execute the request.
-		$request = wp_remote_post( "{$this->api_url}/graphql", $args );
+		$endpoint = self::get_endpoint( 'jobber' );
```

Comment:
This seems to no longer work now. I get a middleware error: `Can't POST to /jobber`.

If I modify this code to hit `/jobber/graphql` it does work again. I think we need to adjust the route on the middleware to only be `/jobber` and not be `/jobber/*`